Solutions/IronNet IronDefense/Playbooks/IronNet_UpdateSentinelIncidents/azuredeploy.json

```diff
@@ -90,42 +51,50 @@
               "defaultValue": {},
               "type": "Object"
             },
-            "IronNetUrl": {
-              "defaultValue": "[parameters('IronDefenseUrl')]",
-              "type": "SecureString"
-            },
-            "Password": {
-              "defaultValue": "[parameters('IronApiPassword')]",
-              "type": "SecureString"
-            },
-            "TenantId": {
-              "defaultValue": "[parameters('TenantId')]",
-              "type": "SecureString"
+            "irondefense_url": {
+							"defaultValue": "[parameters('playbook1-IronDefenseUrl')]",
+							"type": "String"
+						},
+            "tenant_id": {
+              "defaultValue": "[subscription().tenantId]",
+              "type": "String"
             },
-            "Username": {
-              "defaultValue": "[parameters('IronApiUsername')]",
-              "type": "SecureString"
+            "user_name": {
+              "defaultValue": "[parameters('playbook1-IronApiUsername')]",
+              "type": "String"
             },
+            "password": {
+							"defaultValue": "[parameters('playbook1-IronApiPassword')]",
+							"type": "SecureString"
+						},
             "client_id": {
-              "defaultValue": "[parameters('ClientId')]",
-              "type": "SecureString"
+              "defaultValue": "[parameters('playbook1-AzureClientId')]",
```

Comment:
Prameters are missing from the Parameters section, please add them in playbook paramters

Reply:
They are the same as playbook1. Is there possible that I can share the same parameter for playbook1 and playbbok2?